src/components/Studio/Vacancies/Vacancy/Vacancy.module.css

```diff
@@ -0,0 +1,95 @@
+.vacancy {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  padding: 52px 32px 0;
+  background: var(--background-color);
+  z-index: 10;
+  overflow: auto;
+  -webkit-overflow-scrolling: touch;
+  pointer-events: all;
+
+  & :global(button) {
+    border: none;
+    border-radius: 0;
+    background: none;
+  }
+}
+
+.container {
+  max-width: 700px;
+  margin: 0 auto;
+}
+
+.header {
+  @mixin body-3;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 1em;
+
+  @media (--tablet) {
+    @mixin body-1;
+  }
+}
+
+.closeButton,
+.applyButton {
+  outline: none;
+}
+
+.closeButton {
+  @mixin body-2;
+  width: 40px;
+
+  @media (--tablet) {
+    @mixin body-1;
+  }
+}
+
+.applyButton {
+  display: inline-block;
+  margin: 2em auto 3em;
+  color: var(--black) !important;
+  border: 1px solid currentColor;
+  padding: 1em;
+  text-decoration: none;
+}
+
+.description {
+  white-space: pre-wrap;
+
+  p {
+    margin-bottom: 0.5em;
+  }
+
+  em {
+    font-style: normal;
+  }
+
+  h3 {
+    margin: 1em 0 0.5em;
+  }
+
+  ul {
+    list-style: none;
+    margin: 0 0 1em 16px;
+    padding: 0;
+    white-space: normal;
+  }
+
+  li {
+    margin-bottom: 0.25em;
+    &::before {
+      content: '-';
+      position: absolute;
+      letter-spacing: 0.02em;
+      margin-left: -12px;
+    }
+  }
```

Comment:
Normally some of these "resets" ought to be global. Don;t remember whether we do have a reset.css or normalize going on, but may make sense to have some of these in the global stylesheet


e.g. `list-style: none`

Reply:
I don't think we want to do a global list style override, by default a list should behave like a list with bullets etc. This is to fix the specific case of the Vacancy.

Happy to put standardised bullet typography on the Trello, though, so we can make this more global and move out of vacancy?

Reply:
Gotcha, think it's fine as is then unless we'll be having more lists which then needs some standard

Reply:
Added it to Trello, now type sizes are sorted I actually agree would be nice to see wider typography stlyes fixed:
https://trello.com/c/LttrgFii/269-standardise-bullet-typography-styles